GameServerLib/Program.cs

```diff
@@ -17,20 +17,20 @@ public class GameServerLauncher
         public string ConfigJson { get; private set; }
         public ushort ServerPort { get; private set; }
 
-        public GameServerLauncher(ushort serverPort, string configJson)
+        public GameServerLauncher(ushort serverPort, string configJson, string blowfishKey, Action<Logger> onServerStarted)
```

Comment:
Instead of passing an action, do you think we could pass the logger? Would seem more reasonable to me at least.

Reply:
Done but I had to separate the network loop into a function to call that after launching the client. The network loop is a blocking while loop.